test/unit/lodash.spec.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function ()
+{
+    describe('Number functions', function ()
+    {
+        describe('clamp', function ()
+        {
+
+            var params, number, lower, upper;
+
+            before(function ()
+            {
+                params = datasets.clamp();
+                number = params[0];
+                lower = params[1];
+                upper = params[2];
+            });
+
+            it('should verify params', function ()
+            {
+                expect(params).to.have.length(3);
+                expect(lower).to.be.below(upper);
+                expect(typeof number).to.eql('number');
+                expect(typeof lower).to.eql('number');
+                expect(typeof upper).to.eql('number');
```

Comment:
I don't think that we should check types of params because tested method should take only numbers and if sb return in method from datastes sth other then numbers then tests still be like not passing.